modules/cart/js/commerce_cart.js

```diff
@@ -8,19 +8,30 @@
 
   Drupal.behaviors.commerceCartBlock = {
     attach: function (context) {
-      var $context = $(context);
-      var $cart = $context.find('.cart--cart-block');
-      var $cartContents = $cart.find('.cart-block--contents');
+      var $context = $(context),
+          $cart = $context.find('.cart--cart-block'),
```

Comment:
You are talking about $cart?

We use it currently in 2 places. We could change it, but that would things more complicated I guess.